### PR TITLE
docs: update docs with less

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | docs: update docs with less |
| 🇨🇳 Chinese | docs: update docs with less |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/migration-v5.en-US.md [hunk 1]
```diff
@@ -199,21 +199,22 @@ pnpm --package=@ant-design/codemod-v5 dlx antd5-codemod src
...
 const { theme } = require('antd/lib');
+const { convertLegacyToken, defaultTheme } = require('@ant-design/compatible/lib');
 
 const { defaultAlgorithm, defaultSeed } = theme;
 
-const mapToken = defaultAlgorithm(defaultSeed);
-const v4Token = convertLegacyToken(mapToken);
+const mapV5Token = defaultAlgorithm(defaultSeed);
+const v5Vars = convertLegacyToken(mapV5Token);
+const v4Vars = convertLegacyToken(defaultTheme);
```
Comment (MadCcc) [inline docs/react/migration-v5.en-US.md:209]: defaultTheme 里不全的，要用 `theme.getDesignToken()` 算一遍

Comment (crazyair) [reply docs/react/migration-v5.en-US.md]: 漏掉了，改了

File: docs/react/migration-v5.zh-CN.md [hunk 2]
```diff
@@ -193,20 +193,22 @@ pnpm --package=@ant-design/codemod-v5 dlx antd5-codemod src
...
+const { convertLegacyToken, defaultTheme } = require('@ant-design/compatible/lib');
 
 const { defaultAlgorithm, defaultSeed } = theme;
 
-const mapToken = defaultAlgorithm(defaultSeed);
-const v4Token = convertLegacyToken(mapToken);
+const mapV5Token = defaultAlgorithm(defaultSeed);
+const v5Vars = convertLegacyToken(mapV5Token);
+const mapV4Token = theme.getDesignToken(defaultTheme);
+const v4Vars = convertLegacyToken(mapV4Token);
```
Comment (afc163) [inline docs/react/migration-v5.zh-CN.md:203]: 为啥 v5 也要迁移？

Comment (crazyair) [reply docs/react/migration-v5.zh-CN.md:203]: 这个标题是 `如果你使用到了 antd 的 less 变量，通过兼容包将 v5 变量转译成 v4 版本，并通过 less-loader 注入：`
这个是说编译成 v4 的 less 变量，但是值还是 v5 的值，比如 `border-radius-base: 6px` 而不是 `2px`

我这个 pr 增加的是 v4Vars，使用 `defaultTheme` 生成的  less 变量，即 `border-radius-base: 2px`

Comment (crazyair) [reply docs/react/migration-v5.zh-CN.md:203]: 其实使用 less 变量，应该不管 v4 v5，而是可以直接而使用传给 CP 的 theme 生成 less 变量

image

**Review by MadCcc — APPROVED** [review]

**Pull request merged**